NextcloudTalk/BaseChatViewController.swift

```diff
@@ -3318,6 +3318,20 @@ import SwiftUI
             return
         }
 
+        let fileLocalPath = fileParameter.fileStatus?.fileLocalPath ?? ""
+        let fileExtension = URL(fileURLWithPath: fileLocalPath).pathExtension.lowercased()
```

Comment:
```suggestion
        let filePath = fileParameter.path ?? ""
        let fileExtension = URL(fileURLWithPath: filePath).pathExtension.lowercased()
```

`fileStatus` is only available when we start downloading a file, that's why at the moment when you tap on a video the first time, it will start downloading the video file and it will open it in the QLPreviewController. Then the next times it will open it directly in the NCMediaViewer, since it can get the local path from the fileStatus.

If we get the `path` directly from the fileParameter, we are able to check its extension and the video file will be opened in the NCMediaViewer directly.

Reply:
implemented